plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/ConnectionEditor.java

```diff
@@ -169,6 +175,14 @@ public String getText(Object element) {
 		this.rememberTokenCheckbox.setParent(composite);
 		GridDataFactory.fillDefaults()
 				.align(SWT.FILL, SWT.CENTER).span(2, 1).grab(true, false).applyTo(rememberTokenCheckbox);
+		this.rememberPasswordCheckbox.setParent(composite);
+		GridDataFactory.fillDefaults()
+				.align(SWT.FILL, SWT.CENTER).span(2, 1).grab(true, false).applyTo(rememberPasswordCheckbox);
+		
+		((GridData)rememberTokenCheckbox.getLayoutData()).heightHint = 1;
```

Comment:
this looks like it would make the checkbox not taking layouting space which one would normally do via GridData.exclude. There's a util method that does this and the re-layouting in UIUtils.setVisibleAndExclude(boolean visible, final Control control)